backend/src/handlers/alert.py

```diff
@@ -21,52 +20,40 @@ class AlertModel(BaseModel):
 
 
 @router.get("/")
-def get_alerts(
-    req: Request, include: Union[List[str], None] = Query(default=None)
-) -> JSONResponse:
+def get_alerts(req: Request, active: bool = False) -> JSONResponse:
     with req.app.state.db.session() as session:
-        alerts: List[Alert] = session.query(Alert).all()
+        query = session.query(Alert)
+        if active:
+            now = datetime.datetime.now()
+            query = query.filter(Alert.start_datetime <= now, Alert.end_datetime >= now)
+        alerts = query.all()
 
-    resp: Dict[str, List[Dict[str, Union[str, int]]]] = {"alerts": []}
+    alerts_json = []
```

Comment:
Personally, I think we should be strongly typing, and to my understanding, @ezrichards agrees. Let's put but the type.

Reply:
I'll fix this, but we will need to go through and fully type a lot of things in the project, since I haven't been fully typing everything.